dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/hadoop/oozie/AbstractOozieJobsTask.java

```diff
@@ -33,30 +34,34 @@
 import org.apache.commons.beanutils.PropertyUtils;
 import org.apache.commons.csv.CSVFormat;
 import org.apache.commons.csv.CSVPrinter;
-import org.apache.oozie.client.WorkflowJob;
+import org.apache.oozie.client.OozieClientException;
 import org.apache.oozie.client.XOozieClient;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class OozieJobsTask extends AbstractTask<Void> {
-  private static final Logger logger = LoggerFactory.getLogger(OozieJobsTask.class);
+public abstract class AbstractOozieJobsTask<J> extends AbstractTask<Void> {
+  private static final Logger logger = LoggerFactory.getLogger(AbstractOozieJobsTask.class);
   private static final ObjectMapper objectMapper = new ObjectMapper();
 
   private final int maxDaysToFetch;
   private final long initialTimestamp;
+  private final Class<J> oozieJobClass;
 
-  public OozieJobsTask(int maxDaysToFetch) {
-    this(maxDaysToFetch, System.currentTimeMillis());
+  AbstractOozieJobsTask(int maxDaysToFetch) {
+    this("test-flow", maxDaysToFetch, System.currentTimeMillis());
```

Comment:
Why the name here is "test-flow"?

Reply:
This contractor was for test purpose, but I think it's better do delete the contractor for now. 

Deleted